src/main/kotlin/dhbw/karlsruhe/gitgood/controller/Test.java

```diff
@@ -0,0 +1,19 @@
+package dhbw.karlsruhe.gitgood.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class Test {
+
+  @Autowired
+  private JdbcTemplate jdbcTemplate;
+
+  @GetMapping("/test2")
+  public String test() {
+   return jdbcTemplate.queryForObject("SELECT Name FROM Test;", String.class);

```

Comment:
I get an 500 internal server error, when I try to call /test2. Is this supposed to work or am I missing a configuration?

Reply:
This is due to a missing configuration. Thanks for the hint. I will describe the needed configuration in the readme.